ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -118,10 +124,58 @@ def get_demo_files(path: str) -> List[str]:
         )
 
 
+class AgentInfoActionPairProtoDemoGenerator(object):
```

Comment:
```suggestion
class AgentInfoActionPairProtoDemoGenerator:
```
Don't need to inherit explicitly from `object` in python3.